hack/chart/voyager/README.md

```diff
@@ -0,0 +1,46 @@
+# Voyager
+[Voyager](https://github.com/appscode/voyager)  provides controller for Ingress and Certificates for Kubernetes developed by AppsCode.
+## TL;DR;
+
+```bash
+$ helm install stable/voyager
```

Comment:
Use real directory here. so that this command could run. When we move this to kubermetes/charts update there.